### ISSUE TITLE
space at the bottom of the portal

### ISSUE THREAD

**Issue by Eikosa** [remnoteio/remnote-issues#638]
sometimes there is a space under the portal, and sometimes there is no space
![image](https://user-images.githubusercontent.com/20538090/105759307-5bb53200-5f61-11eb-8121-9e14198a55f7.png)
![image](https://user-images.githubusercontent.com/20538090/105759414-81423b80-5f61-11eb-94c6-58d9cb5080e6.png)

I think the cause of this problem is moving portals

**Comment by drkvj** [comment]
Yes I have noticed this a lot of times as well. Thanks for the submit.
